frontend/packages/dev-console/src/components/import/route/SecureRoute.tsx

```diff
@@ -22,6 +17,23 @@ const SecureRoute: React.FC = () => {
     },
     setFieldValue,
   } = useFormikContext<FormikValues>();
+
+  const TerminationTypes = {
+    edge: t('devconsole~Edge'),
+    passthrough: t('devconsole~Passthrough'),
+    reencrypt: t('devconsole~Re-encrypt'),
+  };
+
+  const InsecureTrafficTypes = {
+    None: t('devconsole~None'),
+    Allow: t('devconsole~Allow'),
+    Redirect: t('devconsole~Redirect'),
+  };
+
+  const PassthroughInsecureTrafficTypes = {
+    None: t('devconsole~None'),
+    Redirect: t('devconsole~Redirect'),
+  };
```

Comment:
I should use lower case variable names here. You can also calculate the insecureTraffic options here instead of inline in the JSX.

```suggestion
  const terminationOptions = {
    [TerminationType.edge]: t('devconsole~Edge'),
    [TerminationType.passthrough]: t('devconsole~Passthrough'),
    [TerminationType.reencrypt]: t('devconsole~Re-encrypt'),
  };

  const insecureTrafficOptions = tls.termination === TerminationType.passthrough ? {
    [PassthroughInsecureTrafficType.None]: t('devconsole~None'),
    [PassthroughInsecureTrafficType.Redirect]: t('devconsole~Redirect'),
  } : {
    [InsecureTrafficType.None]: t('devconsole~None'),
    [InsecureTrafficType.Allow]: t('devconsole~Allow'),
    [InsecureTrafficType.Redirect]: t('devconsole~Redirect'),
  };
```